Core/src/Batch/BatchJob.php

```diff
@@ -163,13 +163,24 @@ public function run()
      */
     public function flush(array $items = [])
     {
-        if (!call_user_func_array($this->func, [$items])) {
+        if (! $this->callFunc($items)) {
             $this->handleFailure($this->id, $items);
             return false;
         }
         return true;
     }
 
+    /**
+     * Finish any pending activity for this job.
+     *
+     * @param array $items
+     * @return bool
+     */
+    public function callFunc(array $items = [])
```

Comment:
Is this something we want to be considered part of the public API? Perhaps marking it `@private, @internal` would be preferable.